server/msgset.go

```diff
@@ -357,29 +357,29 @@ func (mset *MsgSet) processInboundJetStreamMsg(_ *subscription, _ *client, subje
 	doAck := !mset.config.NoAck
 	jsa := mset.jsa
 	stype := mset.config.Storage
+	sname := mset.config.Name
+	accName := c.acc.Name
```

Comment:
whats cheaper 1 variable we wont use and later need to GC or getting an additional lock later when we need that? Anyway, seems removing a lock is a good thing so moved that up here

Reply:
`c.acc.Name` surely cant change? So didnt need to be in the lock I think - and was used directly unlocked in the limits check - so I set it outside of the lock now after the c != nil check